PDR-0008-Maintainer-Committee/README.md

```diff
@@ -0,0 +1,52 @@
+---
+PDR: ?
+Title: Maintainer Committees
+Category: Guidelines
+Status: Pending
+Authors:
+    - Christina Gianelloni <christina@blinklabs.io> 
+Created: 2025-02-16
+---
+
+## Context
+The [Statutes](https://github.com/pragma-org/PDRs/blob/main/.incorporation/20240422_Attachment_Statutes_Statuten_PRAGMA_signed.pdf) clearly call out the need for Maintainer Committees and Leads therein. However that information is spread throughout the statutes and at times leaves room for ambiguity. They organize, manage and implement a software project and will need guidance on the expectations and requirements within PRAGMA. 
```

Comment:
Later Maintainer Committee is abbreviated MC, so probably we should write: "Maintainer Committees (MC)" at first occurrence